test/specs/kefir-observable.js

```diff
@@ -0,0 +1,49 @@
+let {Kefir} = require('../test-helpers.coffee');
+
+describe('Kefir.Observable', function() {
+    describe('observe', function() {
+        let em, count, obs, sub;
```

Comment:
Yeah, async tests fail in browser currently. Sorry about that. This is needs to be fixed, but for now you can just ignore them.